client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -505,7 +505,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
             name="owner"
             label={t("terms.owner")}
             fieldId="owner"
-            renderInput={({ field: { value, name, onChange } }) => (
+            renderInput={({ field: { value, name, onChange, onBlur } }) => (
```

Comment:
Is this ever used?

Reply:
@ibolton336 , yes it's used below, line 528.
That's the workaround I found to force the field to be empty in relation of the associated MTA-651 bug.

Reply:
Looks like you're never passing the onBlur. Rather, you are passing onChange as the argument to the onBlur prop for simpleSelect.

Reply:
@ibolton336, gotcha, I thought I was using that onBlur ! I removed it.